Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -216,31 +216,36 @@
     if not args.ttbar:
         addParticleGun(
             s,
+           
             MomentumConfig(
                 args.gun_pt_range[0] * u.GeV,
                 args.gun_pt_range[1] * u.GeV,
                 transverse=True,
             ),
+             
             EtaConfig(args.gun_eta_range[0], args.gun_eta_range[1]),
             PhiConfig(0.0, 360.0 * u.degree),
             ParticleConfig(
                 args.gun_particles, acts.PdgParticle.eMuon, randomizeCharge=True
             ),
-            vtxGen=acts.examples.GaussianVertexGenerator(
+            outputDirRoot=pathlib.Path("/home/aicha/Atlas"),
```

Comment:
The full_chain_odd.py should be kept in its original way. Let's keep the modifications only in the new full_chain_odd_LRT.py